pallets/runtime/common/src/runtime.rs

```diff
@@ -86,15 +79,36 @@ macro_rules! misc_pallet_impls {
             ///
             /// This type is being generated by `construct_runtime!`.
             type PalletInfo = PalletInfo;
+            /// The data to be stored in an account.
+            type AccountData = pallet_balances::AccountData<Balance>;
             /// What to do if a new account is created.
             type OnNewAccount = ();
             /// What to do if an account is fully reaped from the system.
             type OnKilledAccount = ();
-            /// The data to be stored in an account.
-            type AccountData = pallet_balances::AccountData;
             type SystemWeightInfo = polymesh_weights::frame_system::SubstrateWeight;
+            type ExtensionsWeightInfo = ();
```

Comment:
I think we will want to provide weights here.  Most likely we will need to run benchmarks of our transaction extensions.